frontend/packages/knative-plugin/src/utils/kebab-actions.ts

```diff
@@ -21,10 +22,17 @@ const modifyApplicationRefs = [
 ];
 
 export const getKebabActionsForKind = (resourceKind: K8sKind): KebabAction[] => {
+  let menuActions: KebabAction[] = [];
   if (!resourceKind) {
     // no common actions
-    return [];
+    return menuActions;
   }
+  const commonActions = _.includes(modifyApplicationRefs, referenceFor(resourceKind))
+    ? [ModifyApplication]
+    : [];
 
-  return _.includes(modifyApplicationRefs, referenceFor(resourceKind)) ? [ModifyApplication] : [];
+  if (resourceKind.kind === ServiceModel.kind) {
+    menuActions = [setTrafficDistribution];
+  }
+  return [...commonActions, ...menuActions];
```

Comment:
Seems more reasonable to simply push new actions into the array. This will make it easier to add more actions in the future.

```
  const menuActions: KebabAction[] = [];
  if (resourceKind) {
    if (_.includes(modifyApplicationRefs, referenceFor(resourceKind))) {
      menuActions.push(ModifyApplication);
    }
    if (resourceKind.kind === ServiceModel.kind) {
      menuActions.push(setTrafficDistribution);
    }
  }
  return menuActions;
```